examples/rmtchar/rmtchar.h

```diff
@@ -0,0 +1,127 @@
+/****************************************************************************
+ * apps/examples/rmtchar/rmtchar.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __APPS_EXAMPLES_RMTCHAR_RMTCHAR_H
+#define __APPS_EXAMPLES_RMTCHAR_RMTCHAR_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <pthread.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* Configuration */
+
+#ifndef CONFIG_EXAMPLES_RMTCHAR_TX_DEVPATH
+#  define CONFIG_EXAMPLES_RMTCHAR_TX_DEVPATH "/dev/rmt0"
+#endif
+
+#ifndef CONFIG_EXAMPLES_RMTCHAR_RX_DEVPATH
+#  define CONFIG_EXAMPLES_RMTCHAR_RX_DEVPATH "/dev/rmt1"
+#endif
+
+/****************************************************************************
+ * Public Types
+ ****************************************************************************/
+
+struct rmtchar_state_s
+{
+  bool initialized;
```

Comment:
remove, don't need

Reply:
The same of https://github.com/apache/nuttx-apps/pull/2232#discussion_r1430499305